.tool-versions

```diff
@@ -0,0 +1,2 @@
+# make sure to update .github/workflows/publish.yml as well!
+hugo extended_0.124.1
```

Comment:
Mildly sneaking this in, as it was biting me every time I went to work on this repo.